protostar/commands/format/format_command.py

```diff
@@ -88,15 +89,15 @@ async def run(self, args):
             )
 
     def format(
-        self, targets: List[Path], check=False, verbose=False, ignore_broken=False
+        self, targets: List[str], check=False, verbose=False, ignore_broken=False
     ) -> FormattingSummary:
 
         callback: Callable[[FormattingResult], None] = lambda result: print(
             format_formatting_result(result, check)
         )
 
         summary = self._formatter.format(
-            targets=targets,
+            cairo_targets=resolve_cairo_targets(targets),
```

Comment:
```suggestion
            filepaths=resolve_cairo_targets(targets),
```

Reply:
What's the difference between `target` and `cairo_target`?

Reply:
renamed